sugoi-api-jms-utils/src/main/java/fr/insee/sugoi/jms/model/BrokerRequest.java

```diff
@@ -13,23 +13,25 @@
 */
 package fr.insee.sugoi.jms.model;
 
+import java.io.Serializable;
 import java.util.HashMap;
 import java.util.Map;
 
-public class BrokerRequest {
+public class BrokerRequest implements Serializable {
 
-  public String requestBody;
-  public String method;
-  public Map<String, String> queryParams = new HashMap<>();
+  /** */
+  private static final long serialVersionUID = 1L;
 
-  public String getRequestBody() {
-    return this.requestBody;
-  }
+  private String method;
+  private Map<String, Object> methodParams = new HashMap<>();
 
```

Comment:
We could need some more information (like user who made the call, or a request ID for correlation in logs)